### PR TITLE
Add the missing functions in to README.md

### PR DESCRIPTION
**Pull request by 199911**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -23,15 +23,20 @@ npm install slackbots
...
+- `getChannelId` (return: promise) - gets group ID by name,
+- `getGroupId` (return: promise) - gets channel ID by name,
+- `getUserId` (return: promise) - gets user ID by name,
 - `getChatId` (return: promise) - it returns or opens and returns a direct message channel ID,
 - `postMessage` - posts a message to channel | group | user by ID,
 - `postTo(name, message [, params, callback])` - posts a message to channel | group | user by name,
-- `postMessageToChannel(name, message [, params, callback])` - posts a message to channel by name,
 - `postMessageToUser(name, message [, params, callback])` - posts a direct message by user name,
 - `postMessageToGroup(name, message [, params, callback])` - posts a message to private group by name.
+- `postMessageToChannel(name, message [, params, callback])` - posts a message to channel by name,
```
Comment (mishk0) [inline README.md]: Could you specify params and types?

**Comment by 199911** [comment]
I think the current version is good.

As a developer, I want to know what a library can do quickly.
The `Method` section is a good overview, it tells me your library provide the function 
I want, so I decide to use this library and move on the `Usage` section.
The `Usage` section is good, I can find out the parameter type and return value 
from the examples easily.

What make me confusing is, I don't know your library can help me convert user id
to user name until I read the source code. I almost going to write my own version!
So I submit this pull request and want to improve the documentation.

`Method` section as an overview, we better keep it complete and simple.

File: README.md (hunk 1 above)
Comment (mishk0) [inline README.md]: I mean 
```
`getChannelId(name)` (return: promise) - gets group ID by name`
```
It won't take a lot of space, but will be more clear. I think we should keep the same style of the `method` section

**Comment by leviwheatcroft** [comment]
I partly agree with @mishk0, I currently (both in master and in this PR) some of the methods section list params and others don't. It should at least be consistent.

Re: seeing what the library can do quickly, I don't think that adding params is going to increase the time it takes someone to identify what methods are available. 

The method names explain exactly what the methods do anyway, so if you were looking to omit something to save space I'd probably omit the brief description.

**Comment by 199911** [comment]
@mishk0 @leviwheatcroft
That's truth, let me specify params and types first.

File: README.md [hunk 2]
```diff
@@ -22,16 +22,21 @@ npm install slackbots
...
-- `postMessage` - posts a message to channel | group | user by ID,
-- `postTo(name, message [, params, callback])` - posts a message to channel | group | user by name,
-- `postMessageToChannel(name, message [, params, callback])` - posts a message to channel by name,
-- `postMessageToUser(name, message [, params, callback])` - posts a direct message by user name,
-- `postMessageToGroup(name, message [, params, callback])` - posts a message to private group by name.
+- `getChannels()` (return: promise) - returns a list of all channels in the team,
+- `getGroups()` (return: promise) - returns a list of all groups in the team,
+- `getUsers()` (return: promise) - returns a list of all users in the team,
+- `getChannel(name)` (return: promise) - gets group by name,
+- `getGroup(name)` (return: promise) - gets channel by name,
```
Comment (leviwheatcroft) [inline README.md]: sorry to be picky.. I think this description should be swapped with the description for line 28 above.

**Pull request merged**